src/main/java/com/intern/spring/exeptions/CustomNotFoundException.java

```diff
@@ -0,0 +1,22 @@
+package com.intern.spring.exeptions;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public class CustomNotFoundException extends RuntimeException {
+    public CustomNotFoundException(String message) {
```

Comment:
Anh thấy trong các chổ mà em gọi tới class này, chỉ có constructor này được gọi

Reply:
Có cần phải xóa các constructor còn lại không anh, vì em tính viết class này bắt lỗi dùng cho toàn bộ project

Reply:
Cái gì không dùng thì xóa, chưa dùng thì cũng xóa luôn, anh không nghĩ là em cần dùng nhiều vậy đâu

Reply:
fixed